src/components/common/Toolbar/Toolbar.tsx

```diff
@@ -117,6 +139,7 @@ export function Toolbar({ children, ...props }: ToolbarProps) {
       onKeyDown={handleKeyDown}
       disableGutters
       variant="dense"
+      toolbarDirection={toolbarDirection}
```

Comment:
The tests show the following warning:

> Warning: React does not recognize the `toolbarDirection` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `toolbardirection` instead. If you accidentally passed it from a parent component, remove it from the DOM element.

There are different options to solve that. I think the easiest one is to use the `aria-orientation` to decide the `flexDirection` of the component.

Reply:
i already did the lowercase solution

Reply:
Please don't. That is not the solution we want to use!